src/test/functional/queryServices.test/tags.integration.ts

```diff
@@ -444,69 +441,183 @@ describe('tags query tests - happy path', () => {
         }
       }
     `;
+    beforeAll(async () => {
+      getTagsByNameSpy = sinon.spy(TagDataService.prototype, 'getTagsByName');
+      await db('readitla_b.item_grouping').insert([
+        {
+          resolved_id: 1,
+          grouping_id: 10631688,
+          source_score: 1.0,
+        },
+        {
+          resolved_id: 1,
+          grouping_id: 10631689,
+          source_score: 2.0,
+        },
+      ]);
+
+      await db('suggested_tags_user_grouping_tags').insert([
+        {
+          user_id: 1,
+          grouping_id: 10631688,
+          tag: 'zebra',
+          weighted_count: 0.9211,
+          count: 2,
+        },
+        {
+          user_id: 1,
+          grouping_id: 10631689,
```

Comment:
trying to understand this table - does the `grouping_id` groups similar tags? 

and what does the weighted_count and count signify? sorry for so many context question - can do quick zoom call for PR walk through if its easier